src/ORM/Connect/MySQLDatabase.php

```diff
@@ -10,7 +10,6 @@
 use SilverStripe\ORM\DataList;
 use SilverStripe\Model\List\ArrayList;
 use SilverStripe\ORM\DataObject;
-use SilverStripe\ORM\Queries\SQLSelect;
```

Comment:
Looks like this isn't used - I assume you were just removing this to clean up, unrelated to the rest of the PR?
It's fine to do that, but please include a note about it in future so it's immediately clear why the change is being made.

Reply:
Yes, I was just cleaning up.